typescript/sdk/src/ism/metadata/builder.ts

```diff
@@ -132,37 +101,4 @@ export class BaseMetadataBuilder implements MetadataBuilder {
         throw new Error(`Unsupported ISM: ${ism}`);
     }
   }
-
-  static decode(
```

Comment:
This was very problematic and I haven't completely sorted it out so I've lowered the lint severity to `warn` in this folder. 

We shouldn't have a web of static class methods calling each other or cycles will be inevitable. Also it makes usage of inheritance impractical so why use static methods in the first place.